packages/flutter/lib/src/material/outline_button.dart

```diff
@@ -63,39 +47,37 @@ class OutlineButton extends MaterialButton {
     Color color,
     Color highlightColor,
     Color splashColor,
-    double highlightElevation,
     this.borderSide,
     this.disabledBorderColor,
     this.highlightedBorderColor,
     EdgeInsetsGeometry padding,
     ShapeBorder shape,
     Clip clipBehavior = Clip.none,
+    MaterialTapTargetSize materialTapTargetSize,
```

Comment:
The materialTapTargetSize is already provided by the theme. This has some fairly oblique functionality and I don't think it makes sense to customize at the level of an individual button.

Unless of course it somehow isn't retrieving the theme value, then we should fix that instead.

Reply:
`RaisedButton`, `FlatButton`, `FloatingActionButton` and `MaterialButton` all have a `materialTapTargetSize` property. So I think it makes sense that `OutlineButton` gets one as well.

Reply:
ahh you'e right (I wonder when that changed..), carry on!